File: 03. Programming - C#/SchoolHierarchy/Person.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace SchoolHierarchy
{
    public abstract class Person
    {
        private string name;
        public string Name
        {
            get { return name; }
            set { name = value; }
        }
        private int age;
        public int Age
        {
            get { return age; }
            set { age = value; }
        }

    }
}

File: 02. Programming - Java/Homework4/src/exercise/Person.java
package exercise;

public abstract class Person {
	protected String name;
	protected int age;

	public Person(String name, int age) {
		this.name=name;
		this.age=age;
	}

	public String toString() {
		return "Person [name=" + name + ", age=" + age + "]";
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

}
